crates/tick/src/fmt/unix_seconds.rs

```diff
@@ -116,19 +119,27 @@ impl Display for UnixSeconds {
 
 impl From<UnixSeconds> for SystemTime {
     fn from(value: UnixSeconds) -> Self {
-        Self::UNIX_EPOCH + value.0
+        Timestamp::UNIX_EPOCH
+            .saturating_add(value.0)
+            .expect("UnixSeconds value is guaranteed to be within valid range")
+            .into()
     }
 }
 
-impl TryFrom<Duration> for UnixSeconds {
+impl TryFrom<SignedDuration> for UnixSeconds {
```

Comment:
- Hide this method, this exposes jiff `SignedDuration` which we do not want to expose.
- We want to only keep `impl TryFrom<Duration> for UnixSeconds` which will only ever produce UNixSeconds after UNIX epoch